packages/gatsby-theme-carbon/gatsby-config.js

```diff
@@ -50,19 +51,22 @@ module.exports = themeOptions => {
         options: {
           extensions: mdxExtensions,
           gatsbyRemarkPlugins: [
-            { resolve: `gatsby-remark-unwrap-images` },
-            { resolve: `gatsby-remark-smartypants` },
-            {
-              resolve: `gatsby-remark-images`,
-              options: {
-                maxWidth: 1152,
-                linkImagesToOriginal: false,
-                quality: imageQuality,
-                withWebp,
+            ...[
```

Comment:
I could be wrong, but I think this might be why build is failing. You shouldn't need to spread the array or have the array at all.  Maybe try creating the array above outside the return. See other comment

Reply:
Done